cmd/up.go

```diff
@@ -76,6 +80,23 @@ func UpCommand() cli.Command {
 }
 
 func ClusterUp(ctx context.Context, dialersOptions hosts.DialersOptions, flags cluster.ExternalFlags, data map[string]interface{}) (string, string, string, string, map[string]pki.CertificatePKI, error) {
+	// Print proxy related environment variables
```

Comment:
we should move this part to an isolated function and move it outside ClusterUP as its not really related to the clusterup process

Reply:
I suggest to be part of validation.go code

Reply:
I moved it to util, as its not really validating anything, its more that if the authz resources get applied, the proxy in the env vars is used and should be printed. The function is called just before authz resources are being created.